packages/tables/src/Columns/IdColumn.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace Filament\Tables\Columns;
+
+use Filament\Facades\Filament;
+use Illuminate\Database\Eloquent\Model;
+
+class IdColumn extends TextColumn
+{
+    public static function make(string $name = 'id'): static
+    {
+        return parent::make($name);
+    }
+
+    protected function setUp(): void
+    {
+        parent::setUp();
+
+        if ($this->getName() === 'id') {
+            $this->label('ID');
+        }
+
+        $this->sortable();
+
+        $this->url(function (Model $record): ?string {
+            $resource = Filament::getModelResource($record);
```

Comment:
Not all table builder installs have the admin panel installed, so we can't do this check unfortunately

Reply:
Hmm fair, what would you suggest instead @danharrin?

Reply:
We can wrap the entire url() call in a class_exists check if we wanted to.

Reply:
Hmm I don't like that, feels hacky

Reply:
Maybe hacky in a way, but probably still the cleanest solution.